pages/Elevator-Plate.md

```diff
@@ -0,0 +1,7 @@
+## Obtaining
```

Comment:
An article should not start with a heading, always start with an introduction sentence. GitHub already adds a heading for you, so it will look a bit off.